packages/@aws-cdk/aws-glue/test/test.connection.ts

```diff
@@ -0,0 +1,38 @@
+import { expect } from '@aws-cdk/assert';
+import { Stack } from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as glue from '../lib';
+import { Vpc } from '@aws-cdk/aws-ec2';
+
+export = {
```

Comment:
Can you use Jest tests instead of Nodeunit? We're trying to get rid of Nodeunit in our codebase. Should be super easy, just add an (empty) `jest` block to your `package.json` (like this: `"jest": {}`), and then name your tests `connection.test.ts` instead.

Reply:
updated